signalfx-helper.js

```diff
@@ -1,22 +1,23 @@
-'use strict';
+"use strict";
 
-const signalfx = require('signalfx');
-const extractDetailsForSfx = require('./signalfx-transform-helper').extractDetailsForSfx;
+const signalfx = require("signalfx");
+const extractDetailsForSfx = require("./signalfx-transform-helper")
+  .extractDetailsForSfx;
 
-const packageFile = require('./package.json');
+const packageFile = require("./package.json");
 
-const AUTH_TOKEN = process.env.SIGNALFX_AUTH_TOKEN;
+const ACCESS_TOKEN = process.env.SIGNALFX_ACCESS_TOKEN;
 const TIMEOUT_MS = process.env.SIGNALFX_SEND_TIMEOUT;
 
-const INGEST_ENDPOINT = process.env.SIGNALFX_INGEST_ENDPOINT;
+const INGEST_ENDPOINT = process.env.SIGNALFX_METRICS_URL || process.env.SIGNALFX_ENDPOINT_URL;
```

Comment:
iirc in python both `SIGNALFX_AUTH_TOKEN` and `SIGNALFX_INGEST_ENDPOINT` were deprecated w/ warning instead of removed outright

Reply:
I'm planning to release a major version so there should be no surprises. No projects should get auto-upgraded to newer version so this is pretty safe to do IMO.

Reply:
We can add a warning or error in case we find old env vars are uses  and make it crash fast?

Reply:
Done. Added fallback support with deprecation message.